virttest/utils_test/libvirt.py

```diff
@@ -3550,3 +3550,25 @@ def check_qemu_cmd_line(content, err_ignore=False):
             raise exceptions.TestFail("Expected '%s' was not found in "
                                       "qemu command line" % content)
     return True
+
+
+def get_disk_alias(vm, source_file=None):
+    """
+    Get alias name of disk with given source file
+
+    :param vm: VM object
+    :param source_file: domain disk source file
```

Comment:
Add ':return: xxx' as the function has a return value

Reply:
ok, thanks